### PR TITLE
First commit for the Back To Top Button

### PR DESCRIPTION
**Pull request by searchingfor**
Hello,
First time contributing on GitHub. Let me know if I have done anything wrong here, I tried my best to follow the rules. I appreciate any feedback.

Also as a side note, I'm not a fan of how the button hides under the footer once you scroll to the bottom. Wasn't sure how to fix it without messing with the footer.

Thanks!

### CODE REVIEW

File: src/components/BackToTopBtn.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<template>
+  <q-btn
+    v-back-to-top
+    round
+    class="fixed-bottom-right"
+    style="margin: 0 15px 15px 0; background-color: #4a3520"
```
Comment (tiltec) [inline src/components/BackToTopBtn.vue]: Can you move these styles into a scoped class?

Also, I think it's best if you use one of our theme colors, for example by adding `bg-primary` as class.
If none of the theme colors fits, we have to rework our theme ;)

File: src/components/BackToTopBtn.vue [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  QBtn,
+  QIcon,
+  QFixedPosition,
+  BackToTop,
+} from 'quasar'
+export default {
+  components: {
+    QBtn,
+    QIcon,
+    QFixedPosition,
```
Comment (tiltec) [inline src/components/BackToTopBtn.vue]: You don't use QFixedPosition, right? So it could be removed.

File: src/App.vue [hunk 3]
```diff
@@ -11,7 +13,10 @@
  * Root component
  */
 import LoadingProgress from '@/components/Layout/LoadingProgress'
```
Comment (tiltec) [inline src/App.vue]: Better use absolute instead of relative imports, and "@" as alias for "src/" (see line above)

**Comment by searchingfor** [comment]
@tiltec,

What is the best way to get a mobile view of the app?

**Comment by tiltec** [comment]
Depends on your browser, for example:
https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode
https://developers.google.com/web/tools/chrome-devtools/device-mode/

File: config/index.js [hunk 4]
```diff
@@ -53,7 +53,7 @@ module.exports = {
     // auto open browser or not
     openBrowser: true,
     publicPath: '/',
-    port: 8080,
+    port: 3000,
```
Comment (tiltec) [inline config/index.js]: Unrelated. Why did you change the port?

Comment (searchingfor) [reply config/index.js]: My browser was having some kind of glitch and I had to change the port to get it to stop. Sorry about that.

Comment (tiltec) [reply config/index.js]: Please remove before I can merge.
